### ISSUE TITLE
admin/support role [idea]

### ISSUE THREAD

**Issue by sufleR** [netguru/help#93]
I think that it will be good to have some super role (admin/support) to takever task from someone else.

Example:

Task is assigned to person on vacation but it has to be done immediately.
So if I am admin I can take this help task and do it.

what do you think?

**Comment by lubieniebieski** [comment]
you can always do it if you acknowledge the request, right?

**Comment by sufleR** [comment]
no if someone else already acknowledge it.

there is only mark as resolved option.

**Comment by Vojdan-Vrchakovski** [comment]
I agree with you on this one. This kind of feature would be very beneficial
